google-cloud-clients/google-cloud-vision/src/main/java/com/google/cloud/vision/v1/ImageAnnotatorClient.java

```diff
@@ -217,6 +232,117 @@ private final BatchAnnotateImagesResponse batchAnnotateImages(
     return stub.batchAnnotateImagesCallable();
   }
 
+  // AUTO-GENERATED DOCUMENTATION AND METHOD
+  /**
+   * Run asynchronous image detection and annotation for a list of generic files, such as PDF files,
+   * which may contain multiple pages and multiple images per page. Progress and results can be
+   * retrieved through the `google.longrunning.Operations` interface. `Operation.metadata` contains
+   * `OperationMetadata` (metadata). `Operation.response` contains `AsyncBatchAnnotateFilesResponse`
+   * (results).
+   *
+   * <p>Sample code:
+   *
+   * <pre><code>
+   * try (ImageAnnotatorClient imageAnnotatorClient = ImageAnnotatorClient.create()) {
+   *   List&lt;AsyncAnnotateFileRequest&gt; requests = new ArrayList&lt;&gt;();
+   *   AsyncBatchAnnotateFilesResponse response = imageAnnotatorClient.asyncBatchAnnotateFilesAsync(requests).get();
+   * }
+   * </code></pre>
+   *
+   * @param requests Individual async file annotation requests for this batch.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   */
+  public final OperationFuture<AsyncBatchAnnotateFilesResponse, OperationMetadata>
```

Comment:
Well crap I was wondering when this might happen. We have a GA client which formerly only used stable features from GAX, but now the API has added an API method which uses LRO, which isn't stable yet. We need to mark this `@BetaApi` for the moment - I believe we have a visibility field in GAPIC config that can do this.

Reply:
Added `@BetaApi` annotation here manually, and posted [gapic-generator PR](https://github.com/googleapis/gapic-generator/pull/2004) to add it on generation phase for all long-running operations in java.